src/Angor/Client/Pages/Signatures.razor

```diff
@@ -172,43 +157,99 @@
 
                     pendingSignature.AmountToInvest = investorTrx.Outputs.AsIndexedOutputs().Skip(2).Take(investorTrx.Outputs.Count - 3) //Todo get the actual outputs with taproot type
                         .Sum(_ => _.TxOut.Value);
-                    
-                    StateHasChanged();
                 }
                 catch (Exception e)
                 {
                     Console.WriteLine(e);
-                    Console.WriteLine(pendingSignature.TransactionHex);
+                    _Logger.LogDebug(pendingSignature.TransactionHex);
```

Comment:
There should be more text around logs, later we wont be able to know where this came from.